group-coordinator/src/main/java/org/apache/kafka/coordinator/transaction/TransactionLogConfig.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.coordinator.transaction;
+
+public class TransactionLogConfig {
```

Comment:
Hey @OmniaGM. It is a bit weird to have those transaction classes in the group-coordinator module. It does not seem to be the correct place.

Reply:
I feel the same but also not sure where they should move. They don't fit in server module either. I don't see any Jiras to move transaction coordinator out of server but maybe I can start a new module for transaction coordinator similar to the group one. Would this make more sense?

Reply:
Let's see what @dajac says, but I think a `kafka-transactions` or `kafka-transactions-coordinator` module that depends on `kafka-group-coordinator` makes sense to me. The former option is shorter and as clear while the second follows the same pattern as for the group coordinator

Reply:
Yeah, I was using the maven name. I agree that we shouldn't add a dependency unless it exists.

Reply:
I added `transaction-coordinator` modules and moved theses there